.github/workflows/test.yml

```diff
@@ -8,81 +8,42 @@ on:
     paths-ignore:
       - "*.md"
 
-env:
-  PGPASSWORD: postgres
-  TESTDB: greenplum_python_test
-  PGUSER: postgres
-
 jobs:
   build:
-    name: Test Python${{ matrix.python-version }}, PSQL${{matrix.postgres-version}}
+    name: python${{ matrix.python-version }}; ${{ matrix.server }}
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: false
+      fail-fast: true
       matrix:
-        python-version: ["3.9", "3.10"]
-        postgres-version: ['12', '13']
+        python-version: ["3.9", "3.11"]
+        server: ["postgres12-python39", "postgres12-python311"]
```

Comment:
python3.9 and python3.11 pg-3.9 and pg-3.11 these are four different jobs am I right?